tests/redux/reducers/search/searchFiltersReducer-test.js

```diff
@@ -46,6 +46,43 @@ describe('searchFiltersReducer', () => {
         });
     });
 
+    describe('UPDATE_TEXT_SEARCH', () => {
+        const action = {
+            type: 'UPDATE_TEXT_SEARCH',
+            textInput: "testing"
```

Comment:
Please be consistent with the rest of the code base and use single quotes for all strings.